tests/webdav-jest.ts

```diff
@@ -421,30 +423,50 @@ describe('Webdav integration', function testWebdavIntegration() {
     });
   });
 
-  describe('pipeStream(path, stream)', () => {
-    it('should pipe readable streams to the Nextcloud instance', async () => {
+  describe('upload(targetPath, readStream)', () => {
+    it('should pipe data to the Nextcloud instance', async () => {
```

Comment:
Well, it's not wrong that `upload` and `download` both still expect streams, is it?